components/Tutorial/Article.js

```diff
@@ -0,0 +1,335 @@
+    height: '100%',
+    top: 0,
+    left: 95,
+    background: theme.palette.secondary[100],
+    zIndex: -1,
+
+    [theme.breakpoints.down('xs')]: {
+      left: 24,
+    },
+  },
+  subContent: {
+    background: 'white',
+    padding: '20px 40px 28px',
+    borderRadius: 4,
+    marginLeft: 28,
+    whiteSpace: 'pre-line',
+
+    '&:not(:last-child)': {
+      marginBottom: 24,
+
+      [theme.breakpoints.down('xs')]: {
+        marginBottom: 16,
+      },
+    },
+
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: 10,
+      padding: '15px 15px 23px',
+    },
+  },
+  subContentTitleWrapper: {
+    display: 'flex',
+    fontSize: 18,
+    lineHeight: 1.45,
+    fontWeight: 500,
+    marginBottom: 16,
+
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 14,
+      fontWeight: 700,
+    },
+  },
+  subContentLabel: {
+    color: ({ componentTheme }) =>
+      componentTheme === 'yellow' ? theme.palette.primary.main : '#2DAEF7',
+    marginRight: 20,
+
+    [theme.breakpoints.down('xs')]: {
+      marginRight: 30,
+    },
+  },
+  subContentTitle: {
+    letterSpacing: 0.15,
+  },
+  subContentData: {
+    display: 'block',
+    fontSize: 14,
+    letterSpacing: 0.25,
+    lineHeight: 1.45,
+    color: theme.palette.secondary.main,
+
+    [theme.breakpoints.down('xs')]: {
+      fontSize: 12,
+      lineHeight: 1.67,
+    },
+
+    '&:not(:first-child)': {
+      marginTop: 24,
+    },
+  },
+  subContentImage: {
+    maxWidth: '100%',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  subContentLink: {
+    lineBreak: 'anywhere',
+
+    '&:not(:first-child)': {
+      display: 'block',
+      marginTop: 8,
+    },
+  },
+}));
+
+const Article = ({
+  className,
+  label,
+  theme,
+  title,
+  subTitle,
+  content,
+  subContent = [],
+}) => {
+  const subContentRef = useRef();
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const [{ subContentHeight }, updateSpringProps] = useSpring(() => ({
+    subContentHeight:
+      subContentRef.current && isOpen ? subContentRef.current.offsetHeight : 0,
+  }));
+
+  const classes = useStyles({ componentTheme: theme, isOpen });
+
+  const updateSubContentHeight = useCallback(() => {
+    const nextHeight =
+      subContentRef.current && isOpen ? subContentRef.current.offsetHeight : 0;
+
+    updateSpringProps({
+      subContentHeight: nextHeight,
+    });
+  }, [subContentRef, isOpen, updateSpringProps]);
+
+  useEffect(() => {
+    window.addEventListener('resize', updateSubContentHeight);
+
+    return () => window.removeEventListener('resize', updateSubContentHeight);
+  }, [updateSubContentHeight]);
+
+  useEffect(() => {
+    updateSubContentHeight();
+  }, [isOpen, updateSubContentHeight]);
+
+  useEffect(() => {
+    setIsOpen(false);
+  }, [theme, title, subTitle, content, subContent]);
+
+  return (
+    <div className={cx(classes.root, className)}>
+      <div className={classes.label}>
+        <img src={theme === 'yellow' ? yellowLabel : blueLabel} />
```

Comment:
Please consider using existing `<Ribbon>` instead of SVG. You can modify `<Ribbon>` to support blue background.